lib/grppc.gi

```diff
@@ -1852,6 +1852,92 @@ InstallGlobalFunction( GapInputPcGroup, function(U,name)
 
 end );
 
+#############################################################################
+##
+#F  PrintPcPresentation( <grp>, <commBool> )  . . . . . . .  gap input string
+##
+##  Display the pc relations of a pc group.
+##  If <commBool> is true, then the commutator presentation is printed,
+##  otherwise the power presentation.
+##  Trivial commutators / powers are not printed.
+##  The generators are named "g<i>".
+##  The returned boolean indicate if there are commutating generators.
+##
+InstallGlobalFunction( PrintPcPresentation, function(G, commBool)
+    local pcgs, n, F, gens, i, pis, exp, t, h, rel, commPower, j, trivialCommutators;
+
+    pcgs:=Pcgs(G);
+    n:=Length(pcgs);
+    if n=0 then
+        Print("Trivial Group\n");
+        return false;
+    fi;
+    F    := FreeGroup( n, "g" );
+    gens := GeneratorsOfGroup( F );
+    pis  := RelativeOrders( pcgs );
+
+    # the power
```

Comment:
Could you maybe improve this comment so it says more about the following code?

Reply:
I tried to improve it a little. Do you want more?